galaxyui/src/app/my-content/my-content.routing.module.ts

```diff
@@ -36,7 +36,7 @@ const myContentRoutes: Routes = [
         canActivate: [AuthService]
```

Comment:
Seems like canActive should be a part of the app route module.

Reply:
This isn't technically necessary because `canActivate` still protects the routes at the module level. I can add `canLoad` in `app-route.module` which will simply prevent the module from getting loaded at all if the user isn't logged in.